src/converter.bat

```diff
@@ -0,0 +1,4 @@
+wsl make
```

Comment:
what does this converter file convert? what is `wsl` I do not have it on i.e. linux. Can you please add this step to the makefile?

Reply:
My apologies, you don't have to include that file.  It's just a script that I was using that automated making and uploading the hex file.  WSL is Linux incorporated into Windows (windows subsystem Linux). It's so I could compile in Linux.  The pause is so I could check for errors after making the file.

But yeah, you can leave that part out.  If I need to make a new pull request without that in there I can.

Reply:
ok. Then please amend your pullrequest and force-push so we can merge it. Thank you!

Reply:
I made the requested changes.  Thank you!  This is my first time contributing to open source.

Reply:
:+1: